xbmc/GUIInfoManager.cpp

```diff
@@ -4841,7 +4841,7 @@ std::string CGUIInfoManager::GetItemLabel(const CFileItem *item, int info, std::
     break;
   case LISTITEM_EPISODE:
     {
-      int iSeason, iEpisode;
+      int iSeason = 0, iEpisode = 0;
```

Comment:
`iSeason` should probably be initialized to `-1` because `0` means specials (see the check further down).